### PR TITLE
Remove fiber-specific fixtures from 15.6 branch

### PR DESCRIPTION
**Pull request by flarnie**
**what is the change?:**
Removes three directories in the 'fixtures' directory.

**why make this change?:**
These fixtures were designed to work with Fiber, and were accidentally
pulled into this branch. They were causing errors when we try to build
the other fixtures.

**test plan:**
`cd react/fixtures && node ./build-all.js` no longer throws an error
from `fiber-debugger/App.js` - although it still throws another error

**issue:**
https://github.com/facebook/react/issues/9900

### CODE REVIEW

File: fixtures/packaging/babel-standalone.html [hunk 1]
```diff
@@ -1,14 +0,0 @@
-<html>
```
Comment (gaearon) [inline fixtures/packaging/babel-standalone.html]: Do we need to remove this one?

Comment (flarnie) [inline fixtures/packaging/babel-standalone.html]: oop! I didn't mean to remove that. Weird.

Comment (flarnie) [inline fixtures/packaging/babel-standalone.html]: Oh I see - it was nested under 'packaging' and didn't seem like that belonged. I can put it back though.

Comment (gaearon) [inline fixtures/packaging/babel-standalone.html]: Ah, missed that. Yea, fine to remove then.

Comment (gaearon) [inline fixtures/packaging/babel-standalone.html]: I forgot, we added it later. Doesn't make a difference since it tests same thing as `globals`.

**Pull request merged**